applications/tari_app_grpc/proto/wallet.proto

```diff
@@ -31,6 +31,36 @@ import "google/protobuf/timestamp.proto";
 service Wallet {
     // This returns a coinbase transaction
     rpc GetCoinbase (GetCoinbaseRequest) returns (GetCoinbaseResponse);
+    // Send Tari to a number of recipients
+    rpc Transfer (TransferRequest)  returns (TransferResponse);
+}
+
+message TransferRequest {
+    repeated PaymentRecipient recipients = 1;
+}
+
+message PaymentRecipient {
+    string address = 1;
+    uint64 amount = 2;
+    uint64 fee_per_gram = 3;
+    string message = 4;
+}
+
+message TransferResponse {
+  repeated TransferResult results = 1;
+}
+
+message TransferResult {
+    string address = 1;
+    uint64 transaction_id = 2;
+    enum Status {
+        STATUS_PENDING = 0;
+        STATUS_FAILED = 1;
+        STATUS_CANCELLED = 2;
+        STATUS_BROADCAST = 3;
```

Comment:
I don't think this can ever be returned though, same for `STATUS_COMPLETE`

Reply:
The "query a transaction status" method will return these statuses

Reply:
But I could always add those statuses when those calls land in a PR

Reply:
I think it's probably best to keep it concise, it may even be worth creating a new enum for each call and mapping. These protos can be used by any wallet and the implementers may want to know what to do for these enums

Reply:
Done